src/Command/InfectionCommand.php

```diff
@@ -196,7 +196,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $initialTestSuitProcess = $initialTestsRunner->run(
             $testFrameworkOptions->getForInitialProcess(),
             $this->skipCoverage,
-            explode(' ', $input->getOption('initial-tests-php-options'))
```

Comment:
what about checking it with `hasOption()`?

```php
$input->hasOption('initial-tests-php-options') ? $input->getOption('initial-tests-php-options') : $config->getInitialTestsPhpOptions()
```

This will work correcty when `""` is passed as a value of the option. Same below

Reply:
@Fenikkusu are you going to finish this PR or can we take over it?

Reply:
Honestly, I didn't see this come through and hadn't had a chance to circle back to it since. If I get the chance, I can update it tonight. Otherwise you are welcome to run with it.